File: FMAplication/ClientApp/src/app/Shared/Entity/wareHouse/wareHouseReceivedTransfer.ts
import { WareHouse } from "../Inventory";
import { WareHouseReceivedTransferItem } from "./wareHouseReceivedTransferItem";

export class WareHouseReceivedTransfer{
    id: number;
    transactionNumber: string;
    remarks: string;
    isConfirmed: boolean;
    transactionStatus: number;
    transactionDate: string;
    transactionDateStr: string;
    fromWarehouseId: number;
    fromWarehouse: WareHouse;
    toWarehouseId: number;
    toWarehouse: WareHouse;
    sourceTransferId:number;
    items: WareHouseReceivedTransferItem[];
}

File: FMAplication/Domain/WareHouse/WareHouseReceivedTransfer.cs
﻿using FMAplication.Core;
using FMAplication.Enumerations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FMAplication.Domain.WareHouse
{
    public class WareHouseReceivedTransfer:AuditableEntity<int>
    {
        public string TransactionNumber { get; set; }
        public string Remarks { get; set; }
        public bool IsConfirmed { get; set; }
        public TransactionStatus TransactionStatus { get; set; }
        public DateTime TransactionDate { get; set; }
        public int FromWarehouseId { get; set; }
        public int ToWarehouseId { get; set; }
        public WareHouse ToWarehouse { get; set; }
        public int SourceTransferId { get; set; }

    }
}
